--- nano-4.9.2/src/text.c.orig	Sun May 10 17:30:45 2020
+++ nano-4.9.2/src/text.c	Sun May 10 17:32:16 2020
@@ -2362,8 +2362,7 @@
 
 	/* Get the timestamp and the size of the temporary file. */
 	stat(tempfile_name, &fileinfo);
-	timestamp_sec = (long)fileinfo.st_mtim.tv_sec;
-	timestamp_nsec = (long)fileinfo.st_mtim.tv_nsec;
+	timestamp_sec = (long)fileinfo.st_mtime;
 
 	/* If the number of bytes to check is zero, get out. */
 	if (fileinfo.st_size == 0)
@@ -2403,8 +2402,7 @@
 	stat(tempfile_name, &fileinfo);
 
 	/* When the temporary file wasn't touched, say so and leave. */
-	if ((long)fileinfo.st_mtim.tv_sec == timestamp_sec &&
-				(long)fileinfo.st_mtim.tv_nsec == timestamp_nsec) {
+	if ((long)fileinfo.st_mtime == timestamp_sec) {
 		statusbar(_("Nothing changed"));
 		return NULL;
 	}
