--- nano-4.9.2/lib/sys_stat.in.h.orig	Sun May 10 16:52:47 2020
+++ nano-4.9.2/lib/sys_stat.in.h	Sun May 10 16:53:13 2020
@@ -533,42 +533,6 @@
 #endif
 
 
-#if @GNULIB_LSTAT@
-# if ! @HAVE_LSTAT@
-/* mingw does not support symlinks, therefore it does not have lstat.  But
-   without links, stat does just fine.  */
-#  if !(defined __cplusplus && defined GNULIB_NAMESPACE)
-#   define lstat stat
-#  endif
-_GL_CXXALIAS_RPL_1 (lstat, stat, int,
-                    (const char *restrict name, struct stat *restrict buf));
-# elif @REPLACE_LSTAT@
-#  if !(defined __cplusplus && defined GNULIB_NAMESPACE)
-#   undef lstat
-#   define lstat rpl_lstat
-#  endif
-_GL_FUNCDECL_RPL (lstat, int,
-                  (const char *restrict name, struct stat *restrict buf)
-                  _GL_ARG_NONNULL ((1, 2)));
-_GL_CXXALIAS_RPL (lstat, int,
-                  (const char *restrict name, struct stat *restrict buf));
-# else
-_GL_CXXALIAS_SYS (lstat, int,
-                  (const char *restrict name, struct stat *restrict buf));
-# endif
-# if @HAVE_LSTAT@
-_GL_CXXALIASWARN (lstat);
-# endif
-#elif @GNULIB_OVERRIDES_STRUCT_STAT@
-# undef lstat
-# define lstat lstat_used_without_requesting_gnulib_module_lstat
-#elif defined GNULIB_POSIXCHECK
-# undef lstat
-# if HAVE_RAW_DECL_LSTAT
-_GL_WARN_ON_USE (lstat, "lstat is unportable - "
-                 "use gnulib module lstat for portability");
-# endif
-#endif
 
 
 #if @REPLACE_MKDIR@
@@ -709,78 +673,6 @@
 #endif
 
 
-#if @GNULIB_STAT@
-# if @REPLACE_STAT@
-#  if !@GNULIB_OVERRIDES_STRUCT_STAT@
-    /* We can't use the object-like #define stat rpl_stat, because of
-       struct stat.  This means that rpl_stat will not be used if the user
-       does (stat)(a,b).  Oh well.  */
-#   if defined _AIX && defined stat && defined _LARGE_FILES
-     /* With _LARGE_FILES defined, AIX (only) defines stat to stat64,
-        so we have to replace stat64() instead of stat(). */
-#    undef stat64
-#    define stat64(name, st) rpl_stat (name, st)
-#   elif @WINDOWS_64_BIT_ST_SIZE@
-     /* Above, we define stat to _stati64.  */
-#    if defined __MINGW32__ && defined _stati64
-#     ifndef _USE_32BIT_TIME_T
-       /* The system headers define _stati64 to _stat64.  */
-#      undef _stat64
-#      define _stat64(name, st) rpl_stat (name, st)
-#     endif
-#    elif defined _MSC_VER && defined _stati64
-#     ifdef _USE_32BIT_TIME_T
-       /* The system headers define _stati64 to _stat32i64.  */
-#      undef _stat32i64
-#      define _stat32i64(name, st) rpl_stat (name, st)
-#     else
-       /* The system headers define _stati64 to _stat64.  */
-#      undef _stat64
-#      define _stat64(name, st) rpl_stat (name, st)
-#     endif
-#    else
-#     undef _stati64
-#     define _stati64(name, st) rpl_stat (name, st)
-#    endif
-#   elif defined __MINGW32__ && defined stat
-#    ifdef _USE_32BIT_TIME_T
-      /* The system headers define stat to _stat32i64.  */
-#     undef _stat32i64
-#     define _stat32i64(name, st) rpl_stat (name, st)
-#    else
-      /* The system headers define stat to _stat64.  */
-#     undef _stat64
-#     define _stat64(name, st) rpl_stat (name, st)
-#    endif
-#   elif defined _MSC_VER && defined stat
-#    ifdef _USE_32BIT_TIME_T
-      /* The system headers define stat to _stat32.  */
-#     undef _stat32
-#     define _stat32(name, st) rpl_stat (name, st)
-#    else
-      /* The system headers define stat to _stat64i32.  */
-#     undef _stat64i32
-#     define _stat64i32(name, st) rpl_stat (name, st)
-#    endif
-#   else /* !(_AIX || __MINGW32__ || _MSC_VER) */
-#    undef stat
-#    define stat(name, st) rpl_stat (name, st)
-#   endif /* !_LARGE_FILES */
-#  endif /* !@GNULIB_OVERRIDES_STRUCT_STAT@ */
-_GL_EXTERN_C int stat (const char *restrict name, struct stat *restrict buf)
-                      _GL_ARG_NONNULL ((1, 2));
-# endif
-#elif @GNULIB_OVERRIDES_STRUCT_STAT@
-/* see above:
-  #define stat stat_used_without_requesting_gnulib_module_stat
- */
-#elif defined GNULIB_POSIXCHECK
-# undef stat
-# if HAVE_RAW_DECL_STAT
-_GL_WARN_ON_USE (stat, "stat is unportable - "
-                 "use gnulib module stat for portability");
-# endif
-#endif
 
 
 #if @GNULIB_UTIMENSAT@
